docs/Learning-Environment-Design-Agents.md

```diff
@@ -510,20 +510,9 @@ setting the State Size.
 ## Actions
 
 An action is an instruction from the Policy that the agent carries out. The
-action is passed to the Agent as a parameter when the Academy invokes the
-agent's `OnActionReceived()` function. Actions for an agent can take one of two
-forms, either **Continuous** or **Discrete**.
-
-When you specify that the vector action space is **Continuous**, the action
```

Comment:
These sentences are repeated below almost verbatim in their respective sections, so I just removed them.